src/app/maps/m6/directives/synch.directive.ts

```diff
@@ -0,0 +1,31 @@
+import { Directive, ElementRef, HostListener } from '@angular/core';
+
+@Directive({
+  selector: '[appSynch]',
+})
+export class SynchDirective {
+  constructor(private el: ElementRef) {}
+  @HostListener('click')
+  prevFunc() {
+    const slideElements = document.querySelectorAll('.slide-item-v');
+    slideElements.forEach((elem) => {
+      elem.classList.remove('active');
+    });
+
+    this.el.nativeElement.parentElement.parentElement.classList.add('active');
+    const elm = this.el.nativeElement.getAttribute('data-index');
+    const count = document.querySelectorAll('.slide-item.item').length;

```

Comment:
All these manipulations using global styles are super dangerous and not the preferred way to do it in Angular. This should have been a component, using `[class.x]` bindings and `*ngFor`s to craft the elements in the list, instead of doing it 'manually' like this.